app/models/container_image.rb

```diff
@@ -31,9 +31,14 @@ class ContainerImage < ApplicationRecord
 
   acts_as_miq_taggable
   virtual_column :display_registry, :type => :string
+  virtual_column :number_of_containers, :type => :integer
 
   after_create :raise_creation_event
 
+  def number_of_containers
```

Comment:
For a simple count you can use `virtual_total`, no need for `virtual_column` with custom method.  Naming: there is a convention of `total_containers`.